sdk/resourcemanager/azure-resourcemanager-eventhubs/src/main/java/com/azure/resourcemanager/eventhubs/models/EventHubNamespace.java

```diff
@@ -220,6 +235,33 @@ interface WithThroughputConfiguration {
             WithCreate withThroughputUnitsUpperLimit(int units);
         }
 
+        /**
+         * The stage of the event hub namespace definition
+         * allowing to specify minimumTlsVersion.
+         */
+        interface WithMinimumTlsVersion {
+            /**
+             * Set the minimumTlsVersion property: The minimum TLS version for the cluster to support, e.g. '1.2'.
```

Comment:
Improve the javadoc, `Sets the minimum TLC version ...`. Just follow other APIs.

Same for the enableZoneRedundant API javadoc.